lang/en.yml

```diff
@@ -230,9 +230,9 @@ en:
     EMAILDETAILSTAB: 'Email Details'
     PLURALNAME: 'User Defined Form Email Recipients'
     SINGULARNAME: 'User Defined Form Email Recipient'
-    EMAILADDRESSINVALID: 'Email address has a invalid email address'
-    EMAILFROMINVALID: 'Email from has a invalid email address'
-    EMAILREPLYTOINVALID: 'Email reply has a invalid email address'
+    EMAILADDRESSINVALID: 'Email address has an invalid email address'
+    EMAILFROMINVALID: 'Email from has an invalid email address'
+    EMAILREPLYTOINVALID: 'Email reply has an invalid email address'
   UserDefinedForm_EmailRecipientCondition:
```

Comment:
You'll need to escape or use different quotes, but maybe also space our the words:

```yaml
    EMAILADDRESSINVALID: '"Email Address'' is not valid'
    EMAILFROMINVALID: '"Email From" is not valid'
    EMAILREPLYTOINVALID: '"Email Reply To" is not valid'
```